### PR TITLE
Better error message when failing to get Go version

### PR DESCRIPTION
**Pull request by thomshutt**
This addresses https://github.com/Masterminds/glide/issues/481

### CODE REVIEW

**Comment by dmitris** [comment]
lgtm

**Comment by thomshutt** [comment]
@dmitris Any chance of getting this merged?

**Comment by dmitris** [comment]
I'm not the repo admin - @mattfarina is.

File: action/ensure.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func EnsureGoVendor() {
 	// Any release after go15 does not require that env var.
 	cmd = exec.Command(goExecutable(), "version")
 	if out, err := cmd.CombinedOutput(); err != nil {
-		msg.Err("Error getting version: %s.\n", err)
+		msg.Err("Error calling '%s version' (are GOPATH and GOROOT set correctly?): %s.\n", goExecutable(), err)
```
Comment (mattfarina) [inline action/ensure.go]: `GOPATH` and `GOROOT` have defaults now. When would those cause an error getting the version?

Comment (beono) [reply action/ensure.go]: Hello @mattfarina 
I update my golang version from 1.9.1 to 1.10 and then I got this error from glide 
```
glide update
[ERROR] Error getting version: exit status 2.
```
I upgraded golang  in separate terminal so the one that I used with glide had outdated GOROOT path.
I think it's important to update the error message cause it's really confusing at the moment and makes me thinking that something is wrong with my `glide.yaml`

**Comment by thomshutt** [comment]
Doesn't look like this is going to be merged - closing.

**Pull request closed without merging**